app/parse.py

```diff
@@ -1,4 +1,12 @@
+import csv
 from dataclasses import dataclass
+from typing import Any
+
+from bs4 import BeautifulSoup
+import requests
+
+WEBSITE_URL = "http://quotes.toscrape.com"
+TOTAL_PAGES = 10
```

Comment:
it is not a good practice to hardcode the page count
Try to make ur script more universal, u can just check an next button on page